core/src/main/java/org/elasticsearch/monitor/os/OsStats.java

```diff
@@ -187,7 +186,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         }
     }
 
-    public static class Swap implements Writeable, ToXContent {
+    public static class Swap implements Writeable, ToXContentFragment {
```

Comment:
Unrelated to this change, but because I'm just looking at this class: could you re-check CGroup, I think that should also be ToXContentFragment